test/tests/codeclimate_output.rb

```diff
@@ -0,0 +1,17 @@
+class TestCodeClimateOutput < Test::Unit::TestCase
+  def setup
+    @@report ||= Brakeman.run(
+      :app_path       => "#{TEST_PATH}/apps/rails2",
+      :quiet          => true,
+      :run_all_checks => true
+    ).report.to_codeclimate
+  end
+
+  def test_issue_output_is_parseable_as_json
+    issues = @@report.split("\0")
+
+    assert JSON.parse(issues[0]).is_a? Hash
+    assert JSON.parse(issues[1]).is_a? Hash
+    assert JSON.parse(issues[2]).is_a? Hash
```

Comment:
I suggest checking for presence of expeced keys, like in https://github.com/presidentbeef/brakeman/blob/master/test/tests/json_output.rb